install.sh

```diff
@@ -69,9 +69,15 @@ set -euo pipefail
   fi
   echo "bin_path=$bin_path"
 
+  if [[ -n "$version" ]]; then
```

Comment:
```suggestion
  if [[ -n "${version:-}" ]]; then
```
otherwise `set -u` will abort the script because the variable is unset

Reply:
Thank you, I forgot to add `: "${version:=}"` before the if statement